recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - krb5 1.14.*
     - texinfo
     - bison
-    - flex 2.6.0
+    - flex 2.6.3
```

Comment:
I don't recall why we had this pinned to `2.6.0`, but if `2.6.3` works we may as well as leave this unpinned.

Reply:
Okay, I have unpinned flex in my latest commit.  Let's see if it passes CI.